### ISSUE TITLE
Error while building

### ISSUE THREAD

**Issue by ralexrdz** [rowanwins/shp-to-geojson#1]
I get after 
npm install 
npm run build

 Error: Could not resolve './point.js' from src/Shp.js
 
 Would be great to have a browser build as file on the repo
 
 Thanks

**Comment by rowanwins** [comment]
I suspect the error is possibly a windows vs linux thing with filename casing, I'll try and get around to fixing it one day but otherwise a PR is welcome.

The npm package does includes a pre-built browser package
`npm install shp-to-geojson`
and then 
````
import ShpToGeoJson from 'shp-to-geojson/dist/shp-to-geojson.browser.js'
````

**Comment by rowanwins** [comment]
Should be fixed in v1.1.0
